### ISSUE TITLE
Dropping multiple columns

### ISSUE THREAD

**Issue by permanentrance** [JuliaData/DataFramesMeta.jl#275]
Is there a way of dropping multiple columns with `@chain`?

For example, I could chain multiple calls to `select` and that would work:

```
df = DataFrame(a = repeat(1:4, outer = 2), b = repeat(2:-1:1, outer = 4), c = 1:8);
@chain df begin
  select(Not(:a))
  select(Not(:b))
end
```

But I was wondering if there is a cleaner way of doing it, maybe with `@select` ?

**Comment by pdeffebach** [comment]
Yes, you can wrap it in `cols`, i.t. `@select(df, cols(Not(:a))`

```
julia> @chain df begin
         @select(cols(Not(:a)))
         @select(cols(Not(:b)))
       end
8×1 DataFrame
 Row │ c
     │ Int64
─────┼───────
   1 │     1
   2 │     2
   3 │     3
   4 │     4
   5 │     5
   6 │     6
   7 │     7
   8 │     8
```

This is ugly syntax, but soon `cols` will be replaced by `$`, so it will look less noisy. The PR has already been merged into master [here](https://github.com/JuliaData/DataFramesMeta.jl/pull/266).

**Comment by bkamins** [comment]
Also note you can just write `Not([:a, :b])` if this is something you are looking for.

In DataFrames.jl we try to ensure that what provide is fully composable. Here - it means that you can throw ANY column selector as an argumen to `Not`.

**Comment by pdeffebach** [comment]
ref #283

**Comment by pdeffebach** [comment]
Closed to focus on #283
